pinery-ws-dto/src/main/java/ca/on/oicr/ws/dto/Dtos.java

```diff
@@ -536,5 +538,9 @@ public static BoxPositionDto asDto(BoxPosition from) {
       to.setSampleId(from.getSampleId());
       return to;
     }
+    
+    private static String format(Date date) {
+        return ZonedDateTime.ofInstant(Instant.ofEpochMilli(date.getTime()), ZoneId.systemDefault()).format(DateTimeFormatter.ISO_OFFSET_DATE_TIME);
```

Comment:
`.truncatedTo(ChronoUnit.SECONDS)` before format will fix tests and keep the format consistent